focus-group.js

```diff
@@ -28,13 +32,25 @@ export function focusGroupKeyUX(options) {
       }
     }
 
+    function getItems(eventTarget, group) {
+      if (group.role === "toolbar") return getToolbarItems(group)
+      return group.querySelectorAll(`[role=${eventTarget.role}]`)
+    }
+
+    function getToolbarItems(group) {
+      let items = [...group.querySelectorAll('*')]
+      return items.filter((item)=> {
```

Comment:
Format is not according to Prettier

Reply:
Previously I used only **Editor Config** for the project should I install **Prettier** auto formatter ?

Reply:
Editor Config is for general settings (indent, new line format, etc).

Prettier is about formatting the code, where to add spaces, etc.

I recommend installing it since it is an industry standard nowadays.

But you can keep code formatting manually if you don’t want to install it.

Reply:
formated

Reply:
Prettier is awesome I just clarified the info